lib/styles/theme.js

```diff
@@ -0,0 +1,29 @@
+export default {
+  colors: {
+    white: '#FFFFFF',
+    neutral: {
+      dark: 'rgb(41, 51, 61)',
+      base: 'rgb(133, 153, 173)',
+      light: 'rgb(224, 229, 234)',
+      lightest: 'rgb(249, 250, 251)'
+    }
+  },
+  fonts: {
+    sizes: {
+      xlarge: '1.5rem',
+      large: '1.25rem',
+      lead: '1.125rem',
+      base: '1rem',
+      slight: '.875rem',
+      small: '.75rem',
+      micro: '.6875rem'
+    },
+    lineHeights: {
+      base: '1.25rem',
+      lead: '1.5rem'
+    }
+  },
+  layout: {
+    spacingBase: 20
```

Comment:
We've been wanting this for a while in `scss` bu never added it - lets bake in some modifiers by default like `half` or whatever, what do you think?

Reply:
Great idea, I'll add those in :+1: